### PR TITLE
Add notebook.

### PR DESCRIPTION
**Pull request by mokkiebear**
## Задача
Cсылка в Jira GPN на задачу: https://jira.konakov.tv/browse/VEGA-6125

## Problem statement

Добавить записную книжку, которую можно будет использовать в микрофронтах.
Добавить типы для записной книжки.

## Solution details

_{What has been done (list of details)}_

## Type
- [x] Feature
- [ ] Bug Fix
- [ ] Refactoring
- [ ] Test
- [ ] Other

## Associated issues

_{Jira issues or PRs related to this one}_

## Quality control
- [x] PR: Based on a correct branch
- [x] PR: Head branch is rebased on the base branch
- [x] PR: Assignee chosen and necessary labels are set
- [x] PR: Current form is filled out (where it makes sense)
- [x] PR: Diff checked for irrelevant changes
- [x] PR: Commit messages follow [the rules](../docs/commits-style.md)
- [x] JS: There's no errors/warnings in the browser dev console
- [x] Layout: Uses [variables](https://github.com/gpn-prototypes/ui-kit/tree/master/src/components/Theme) and [CSS classes](https://github.com/gpn-prototypes/ui-kit/blob/master/src/utils/whitepaper/whitepaper.css) from [GPN design system](https://github.com/gpn-prototypes/ui-kit)
- [ ] Layout: Tested with various content amounts
- [ ] Docs: All changes to the component API are reflected and key points are described
- [ ] Storybook: Stories are written/updated for components
- [ ] Storybook: Stories follow [the rules](https://github.com/gpn-prototypes/vega-ui/blob/master/docs/storybook.md)
- [ ] Tests: New features and bugfixes are covered with tests
- [ ] Tests: Existing tests passed successfully

## Notable statements
- [ ] Affects configuration files
- [ ] Brings new packages or updates existing
- [ ] Opened follow-up tasks to resolve

### CODE REVIEW

**Review by ndrzhn — APPROVED** [review]

File: src/types/notebook.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  FrontendNotebookConception
+>;
+
+export interface FrontendNotebookConception {
+  rbColumns: Record<string, FrontendNotebookRbColumn>;
+}
+
+export interface FrontendNotebookRbColumn {
+  frozen: boolean;
+}
```
Comment (ndrzhn) [inline src/types/notebook.ts:50]: а где описан тип самого сервиса? а то в модулях не видно его методы

**Review by ndrzhn — CHANGES_REQUESTED** [review]
написала коммент

File: src/services/notebookService.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import {
+  FrontendNotebookData,
+  GetFrontendNotebookResponse,
+  SaveFrontendNotebookResponse,
+} from 'src/types';
+
+interface Params {
+  getToken: () => Promise<string | null>;
```
Comment (ndrzhn) [inline src/services/notebookService.ts:8]: здесь точно null нужен?

Comment (ndrzhn) [reply src/services/notebookService.ts:8]: нужен

Comment (mokkiebear) [reply src/services/notebookService.ts:8]: Думаю да

**Review by SergeyKatugin — APPROVED** [review]

**Review by ndrzhn — APPROVED** [review]

**Review by SergeyKatugin — APPROVED** [review]

**Pull request merged**